terrabrasilis.js

```diff
@@ -1847,10 +1847,15 @@ Terrabrasilis = (function () {
     $(dom).loading('stop')
   }
 
-
   const fitBounds = function (layerMetada) {
-    return utils.getBounds(layerMetada).then((bounds) => {
-      map.fitBounds(bounds)
+    return new Promise((resolve, reject) => {
+      utils.getBounds(layerMetada)
+        .then((bounds) => {
+          map.fitBounds(bounds)
+          resolve()
+        })
+        .catch(reject)
+        .finally(resolve)
```

Comment:
Not sure if it is effective to call the method finally in this solution, because resolution or rejection methods will be called before finally, so your callbacks no longer expect responses.

Reply:
That's true, that was a trash from the tests I was doing, I'm gonna fix it ASAP